### DISCUSSION TITLE
Is there another option to force StaxRip to use more cores than Chunks?

### DISCUSSION THREAD

**Discussion by ZeroCool22** [staxrip/staxrip#573] [Q&A]
I read something about a **"Threads"** option for x265, but can't find it.

![Screenshot_6](https://user-images.githubusercontent.com/13344308/110379695-e8114500-8035-11eb-9950-41dd5dc4fdf4.png)

Also this guy said using Chunks can **affect the final quality**, it's truth?

**Comment by Dendraspis** [comment]
1. Don't believe everything somebody wrote in a forum

2. Use the [x265 docs](https://x265.readthedocs.io/en/master/cli.html) to check if a statement could be right.

3. x265 doesn't have a parameter called `--threads` - check the docs!

4. Maybe he means `--frame-threads`, but the docs say:
`Default is an autodetected count based on the number of CPU cores and whether WPP is enabled or not.`
`Over-allocation of frame threads will not improve performance, it will generally just increase memory use.`
`Values: any value between 0 and 16. Default is 0, auto-detect`
So there is no benefit.

> Also this guy said using Chunks can affect the final quality, it's truth?

Yes and no. It can affect the quality under some conditions, but you would surely never notice it. Even if somebody could notice it, it would affect just approx. half a second before a cut.

### Conclusion:
The best way is to run multiple parallel encodes. If you have just one encode, that you want to finish ASAP, use Chunks.

**Reply by ghost** [reply]
OMG the kind of lies you can get on the internet!

1. @Dendraspis  is saying the truth, you must listen to him, because ALL he said is correct.
2. the only situation where you can have degradation of quality is just before the cut point (as Dendraspis said) AND ONLY IF YOU ARE USING VERY LOW BIT RATES AND PRESETS FASTER THAN FAST OR EVEN VERY FAST

**Reply by ZeroCool22** [reply]
Thx.

Will leave it here: https://forums.whirlpool.net.au/archive/3jwn41z9 just for you to laugh. xD
